doc/index.rst

```diff
@@ -185,6 +185,7 @@ Note that some of these currently tied only to a single option, but we still lea
 * **DOCKER_RUN_OPTS** (default: not set): Used to specify additional run options for Docker.
 * **DOWNSTREAM_CMAKE_ARGS** (default: not set): Addtional CMake arguments for downstream `workspace <#workspace-management>`__.
 * **DOWNSTREAM_WORKSPACE** (default: not set): Definition of downstream `workspace <#workspace-management>`__.
+* **EXTEND_FROM** (default: ``/opt/ros/$ROS_DISTRO``): Path to a workspace to be used as an underlay of the workspaces being set up be ICI, e.g. a workspace provided by a custom docker image
```

Comment:
We could just call it `UNDERLAY`.
"extend" was meant as "extend the chain", but it might not make much sense to users.